Helper/PaymentMethods.php

```diff
@@ -307,8 +307,10 @@ protected function getCurrentCountryCode($store, $country = null)
             return $country;
         }
 
-        if ($country = $this->getQuote()->getBillingAddress()->getCountry()) {
-            return $country;
+        $quote = $this->getQuote();
+        $customerId = $quote->getCustomerId();
+        if (isset($customerId)) {
```

Comment:
```suggestion
        if (!is_null($quote->getCustomer()) && !is_null($quote->getCustomer()->getDefaultBillingAddress())) {
```

Default billing address may be null for a customer if she/he didn't fill it before.
In that case, `$quote->getCustomer()->getDefaultBillingAddress()` will throw an exception.